app/build.gradle

```diff
@@ -121,6 +125,8 @@ dependencies {
     implementation "androidx.preference:preference:$PREFERENCE_VERSION"
     // Kotlin
     implementation "androidx.preference:preference-ktx:$PREFERENCE_VERSION"
+
+    implementation 'androidx.multidex:multidex:2.0.1'
```

Comment:
I had to enable multidex support as the number of methods grew after adding a couple of dependencies.